python/foglamp/common/configuration_manager.py

```diff
@@ -426,6 +426,71 @@ def get_entry_val(k):
             err_response = ex.error
             raise ValueError(err_response)
 
+    async def update_configuration_item_bulk(self, category_name, config_item_list):
+        """ Bulk update config items
+
+        Args:
+            category_name: category name
+            config_item_list: dict containing config item values
+
+        Returns:
+            None
+        """
+
+        try:
+            payload = {"updates": []}
+            audit_details = {'category': category_name, 'items': {}}
+            for item_name, new_val in config_item_list.items():
+                if not isinstance(new_val, str):
```

Comment:
Allow dict for JSON

Reply:
actually, _validate_type_value is already there hence you can skip this check at all.

Reply:
> actually, _validate_type_value is already there hence you can skip this check at all.

Nope, we should restrict before if the expected value is not in string format. JSON is an exceptional case.

Reply:
I'm not sure what is the value of checking twice. what is the use of _validate_type_value

Reply:
**_validate_type_value** converts string to type as given. If value not "string" then it raises an Exception

Reply:
Yeah, that is good enough!